ideascube/db_router.py

```diff
@@ -1,30 +1,44 @@
+
+_databaseToUse = {
+    ('search', 'search') : 'transient'
```

Comment:
Using a tuple as a key works, but is not super elegant IMHO.

Why not model names as keys of a dict for the app?

```python
_databaseToUse = {
    'search': {
        'search': 'transient',
    }
}
```